docs/explainers/Architecture.md

```diff
@@ -90,6 +90,8 @@ _There are currently 9 file types that make up a cap table_:
    storing vesting schedules used by the issuer.
 9. [Financings File(s)](../schema_markdown/schema/files/FinancingsFile.md) - One or more files
    storing the issuer's financing events.
+10. [Documents File(s)](../schema_markdown/schema/files/DocumentsFile.md) - One or more files
+    storing references to related stakeholder documents.
```

Comment:
Let's also add some brief documentation on how the document package would work here. If it's riding alongside the normal ocf file in a separate zip, we should describe this.

Reply:
Updated, please review. Thanks!